pkg/api/testapi/testapi.go

```diff
@@ -322,14 +327,13 @@ func (g TestGroup) SelfLink(resource, name string) string {
 			return fmt.Sprintf("/api/%s/%s", g.externalGroupVersion.Version, resource)
 		}
 		return fmt.Sprintf("/api/%s/%s/%s", g.externalGroupVersion.Version, resource, name)
-	} else {
-		// TODO: will need a /apis prefix once we have proper multi-group
-		// support
-		if name == "" {
-			return fmt.Sprintf("/apis/%s/%s/%s", g.externalGroupVersion.Group, g.externalGroupVersion.Version, resource)
-		}
-		return fmt.Sprintf("/apis/%s/%s/%s/%s", g.externalGroupVersion.Group, g.externalGroupVersion.Version, resource, name)
 	}
+	// TODO: will need a /apis prefix once we have proper multi-group
```

Comment:
is it necessary for golint?

Reply:
golint reported this: 
`pkg/api/testapi/testapi.go:325:9: if block ends with a return statement, so drop this else and outdent its block`
Which makes sense, right? If we have return in if itself, then there is no need of else. We can just keep the logic outside of else block.